docs/resources/vpc_connect.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: "cloudamqp"
+page_title: "CloudAMQP: cloudamqp_vpc_connect"
+description: |-
+  Enable VPC connect (Privatelink or Private Service Connect) for a CloudAMQP instance hosted in
+  AWS, Azure or GCP.
+---
+
+# cloudamqp_vpc_connect
+
+This resource is a generic way to handle PrivateLink (AWS and Azure) and Private Service Connect (GCP).
+Communication between resources can be done just as they were living inside a VPC. CloudAMQP creates an Endpoint
+Service to connect the VPC and creating a new network interface to handle the communicate.
+
+If no existing VPC available when enable VPC connect, a new VPC will be created with subnet `10.52.72.0/24`.
+
+More information can be found at: [CloudAMQP privatelink](https://www.cloudamqp.com/docs/cloudamqp-privatelink.html)
+
+-> **Note:** Enabling VPC Connect will automatically add a firewall rule.
+
+<details>
+ <summary>
+    <b>
+      <i>Default PrivateLink firewall rule [AWS, Azure]</i>
+    </b>
+  </summary>
+```hcl
+rules {
+  Description = "PrivateLink setup"
+  ip          = "<VPC Subnet>"
+  ports       = []
+  services    = ["AMQP", "AMQPS", "HTTPS", "STREAM", "STREAM_SSL", "STOMP", "STOMPS", "MQTT", "MQTTS"]
+}
+```
+</details>
+
+<details>
+ <summary>
+    <b>
+      <i>Default Private Service Connect firewall rule [GCP]</i>
+    </b>
+  </summary>
+```hcl
```

Comment:
Looks like we need a blank line here for this to render correctly (at least on GitHub, see https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/eb7dc0e486ff12e47543b39bbda493afdf70e1ae/docs/resources/vpc_connect.md) We use blank line further down in this document

Reply:
Nice find. Using https://registry.terraform.io/tools/doc-preview when writing them. But should be doubled checked against github's markdown too.